apps/prairielearn/src/pages/studentAssessment/studentAssessment.ts

```diff
@@ -34,10 +34,10 @@ const router = Router({ mergeParams: true });
 router.use(selectAndAuthzAssessment);
 router.use(studentAssessmentRedirect);
 router.use(studentAssessmentAccess);
-router.use(logPageView('studentAssessment'));
 
 router.get(
   '/',
+  logPageView('studentAssessmentInstance'),
```

Comment:
Internally, this already only runs for GET requests; I felt it was clearer to put this just on the `GET` handler. Similar for other files in this PR.